applications/nrf5340_audio/src/modules/audio_i2s.c

```diff
@@ -69,6 +69,16 @@ static enum audio_i2s_state state = AUDIO_I2S_STATE_UNINIT;
 
 PINCTRL_DT_DEFINE(I2S_NL);
 
+#if (CONFIG_AUDIO_SAMPLE_RATE_HZ == 16000)
```

Comment:
Maybe better to use AUDIO_SAMPLE_RATE_16000_HZ etc. here?

Reply:
Fixed